e2e/single-cluster/single_cluster_test.go

```diff
@@ -59,6 +60,27 @@ var _ = Describe("Single Cluster Examples", func() {
 			})
 		})
 
+		Context("containing a private oci based helm chart", func() {
+			BeforeEach(func() {
+				asset = "single-cluster/helm-oci-with-auth.yaml"
+				k = env.Kubectl.Namespace(env.Namespace)
+
+				out, err := k.Create(
+					"secret", "generic", "helm-oci-secret",
+					"--from-literal=username="+os.Getenv("CI_OCI_USERNAME"),
+					"--from-literal=password="+os.Getenv("CI_OCI_PASSWORD"),
+				)
+				Expect(err).ToNot(HaveOccurred(), out)
+			})
+
+			It("deploys the helm chart", func() {
+				Eventually(func() string {
+					out, _ := k.Namespace("fleet-helm-oci-with-auth-example").Get("pods")
+					return out
+				}, testenv.Timeout).Should(ContainSubstring("frontend-"))
```

Comment:
```suggestion
				}).Should(ContainSubstring("frontend-"))
```
It's the default value.